crates/query-engine/translation/src/translation/mutation/generate.rs

```diff
@@ -1,35 +1,33 @@
 //! Given introspection data, generate a set of standard mutation procedures
 
-use super::delete::{generate_delete_by_unique, DeleteMutation};
-use super::insert;
-use super::insert::InsertMutation;
 use query_engine_metadata::metadata::{database, mutations};
 use std::collections::BTreeMap;
 
+use super::experimental;
+use super::v1;
+
 #[derive(Debug, Clone)]
 pub enum Mutation {
-    DeleteMutation(DeleteMutation),
-    InsertMutation(InsertMutation),
+    V1(v1::Mutation),
+    Experimental(experimental::Mutation),
```

Comment:
Shouldn't this also be called `VeryExperimentalWip`?

Reply:
The name of the flag is deliberately exaggerated, but we don't have to do that internally :)

I assume we won't want to have two distinct versions at the same time, but if we do, we can give them different names, sure.
This `VeryExperimentalWip` version should be renamed to `V2` once we fix it after all.